### PR TITLE
1231 Update Footer when page content is shorter than vh

### LINKED ISSUE #1231: Fix Privacy Policy footer placement
### Overview

The footer on the Privacy Policy doesn't reach the bottom of the browser window. 

Screenshot 2024-03-11 at 10 27 30 AM

### Action Items

-  [x] fix the footer so it appears at the bottom of the browser window even when content doesn't reach

### PR DESCRIPTION
**Pull request by eburdekin**
Fixes #1231 

### Changes made
- Move AppHeader, AppSubheader, and AppFooter components into PageContainer in App.tsx
- Adjust PageContainer minHeight to 100vh in utilityStyles.ts with adjustment for sm breakpoint

### Reason for changes
- The footer on the Privacy Policy doesn't reach the bottom of the browser window. I also noticed it doesn't reach the bottom on some steps of the Letter Generator so included a few screenshots of those as well. To be honest I'm not sure this is the most elegant solution with the setting for the sm breakpoint, but I couldn't quite crack it without moving those layout components inside the PageContainer. Very open to feedback!

<details>
<summary>Visuals before changes applied</summary>

Screen Shot 2024-04-08 at 3 06 47 PM
Screen Shot 2024-04-08 at 3 07 10 PM
Screen Shot 2024-04-08 at 3 07 52 PM
Screen Shot 2024-04-13 at 12 08 45 AM
Screen Shot 2024-04-13 at 12 09 44 AM

</details>

<details>
<summary>Visuals after changes applied</summary>

Screen Shot 2024-04-08 at 3 06 54 PM
Screen Shot 2024-04-08 at 3 07 20 PM
Screen Shot 2024-04-08 at 3 07 59 PM
Screen Shot 2024-04-13 at 12 08 28 AM
Screen Shot 2024-04-13 at 12 09 31 AM

</details>

### CODE REVIEW

File: products/statement-generator/src/styles/utilityStyles.ts [hunk 1]
```diff
@@ -4,13 +4,17 @@ const useUtilityStyles = makeStyles<Theme>(
...
         width: '100%',
         display: 'flex',
         flex: '1 0 auto',
         flexDirection: 'column',
         position: 'relative',
 
+        [breakpoints.down('sm')]: {
+          minHeight: 'calc(100vh + 155px)',
+        },
+
```
Comment (sydneywalcoff) [inline products/statement-generator/src/styles/utilityStyles.ts:17]: what is this intended to do? It currently is pushing the footer off the screen on mobile screens.

**Review by sydneywalcoff — COMMENTED** [review]
@eburdekin Great solution! I have a question about  the breakpoint you added, but I love your use of existing code to solve this

File: products/statement-generator/src/styles/utilityStyles.ts (hunk 1 above)
Comment (sydneywalcoff) [reply products/statement-generator/src/styles/utilityStyles.ts:17]: @eburdekin let's remove the breakpoint and leave it as it was. It's not on us to make design decisions and this creates a scroll on pages that don't require it which is a UX thing.

Comment (sydneywalcoff) [reply products/statement-generator/src/styles/utilityStyles.ts:17]: You can leave how you adjusted the PageContainer minHeight to 100vh in utilityStyles.ts, just remove the adjustment for sm breakpoint

**Comment by eburdekin** [comment]
Thanks again for the feedback @sydneywalcoff ! Updated to remove those lines from utilityStyles.ts.

**Review by sydneywalcoff — APPROVED** [review]

**Pull request merged**